exercises/practice/change/.meta/proof.ci.wren

```diff
@@ -17,34 +20,33 @@ class Change {
    *
    * S = the _first_ coin used to make change for amount n.
    */
-  static change(d, n) {
-    var C = [0] + List.filled(n, Num.largest)
-    var S = [0] + List.filled(n, null)
+  static change(denominations, valueOfCoins) {
+    var C = [0] + List.filled(valueOfCoins, null)
+    var S = [0] + List.filled(valueOfCoins, null)
 
-    var p = 1
-    while (p <= n) {
-      var min = Num.largest
-      var coin = null
-      for (c in d) {
+    for (p in 1..valueOfCoins) {
```

Comment:
Since I didn't exit early for target == 0, I couldn't use `(1..valueOfCoins).each ...` as wren allows descending ranges. Hence the `while` loop.

Reply:
Yeah first thing I ran into.  I think this is much nicer than requiring the extra intermediate variable and `target == 0` also smells like enough of a real edge case to warrant it [being more explicit].